src/flex/FlexView.js

```diff
@@ -43,6 +43,7 @@ export default React.createClass({
       React.PropTypes.number,
       React.PropTypes.string
     ]),
+    events: React.PropTypes.object,
```

Comment:
it is, but I wanted to be clear that it is intended to be used only for events as it will passed to first child which is a react `div`. I don't want it to be used for other purposes like `style`, `className` or `id`. Does react `div` accept non events props other than the three above?